flink-connectors/flink-connector-hive/pom.xml

```diff
@@ -1024,9 +1024,25 @@ under the License.
 	<profiles>
 		<!-- Activate these profiles with -Phive-x.x.x to build and test against different Hive versions -->
 		<profile>
-			<id>hive-3.1.1</id>
+			<id>hive-3.1.2</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.apache.maven.plugins</groupId>
+						<artifactId>maven-surefire-plugin</artifactId>
+						<configuration>
+							<!-- disable running test in parallel for hive 3.1.2, otherwise, there'll be some concurrency issues -->
```

Comment:
Do you have an example for such an issue?

Reply:
It would be good to figure out what the underlying issue is.
Because given that the tests do not run concurrently within a single JVM I would have to conclude that they share some data on disk, which is a terrible idea to start with.
We just spent quite some effort to get things to run with multiple JVMs and re-using forks; we are not throwing that away for Hive 3.

Reply:
This is not an issue with concurrency. It fails when multiple hive tests are run sequentially within the same JVM. This implies something isn't being cleaned up properly when the test exits; and a subsequent test picks up the empty database.

Reply:
@zentol Thanks very much for your detail explaination and sorry for my misleading. After spending some time debuging, I found the root cause.  Put it simply, the reason is some test modify a static variable, and then boil the other test.

More detail is in [FLINK-27999](https://issues.apache.org/jira/browse/FLINK-27999).  I'll fix [FLINK-27999](https://issues.apache.org/jira/browse/FLINK-27999) firstly. After it merges, I think we revisit this pr.

Reply:
Perfect, thanks for looking into it 👍